apps/emqx/test/emqx_persistent_messages_SUITE.erl

```diff
@@ -205,31 +205,7 @@ t_session_subscription_iterators(Config) ->
                 messages => [Message1, Message2, Message3, Message4]
             }
         end,
-        fun(Trace) ->
-            ct:pal("trace:\n  ~p", [Trace]),
-            case ?of_kind(ds_session_subscription_added, Trace) of
```

Comment:
This postcondition was verifying a property that is now assured by the `persistent_session_ds_state` module. Removed it to simplify the tests.